client/header/activity-panel/panels/reviews.js

```diff
@@ -22,62 +23,18 @@ import {
 	SplitButton,
 } from '@woocommerce/components';
 
-// TODO Pull proper data from the API
-const demoReviews = [
-	{
-		id: 1,
-		product_id: 1,
-		reviewer: 'Justin Shreve',
-		reviewer_email: 'justin@automattic.com',
-		rating: 3,
-		verified: true,
-		review:
-			'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque finibus hendrerit finibus.' +
-			'Integer tristique turpis a aliquam aliquam. Phasellus sapien lectus, sodales in sagittis nec, placerat a augue.',
-		status: 'pending',
-		date_created: '2018-07-10T02:49:00Z',
-	},
-];
-
-const demoProducts = [
-	{
-		id: 1,
-		name: 'WordPress Shirt',
-		permalink: '#',
-		images: [
-			{
-				id: 1,
-				src: 'https://cldup.com/5QyaPgyfFo-3000x3000.png',
-			},
-		],
-	},
-];
+/**
+ * Internal dependencies
+ */
+import { ActivityCard, ActivityCardPlaceholder } from '../activity-card';
+import ActivityHeader from '../activity-header';
+import { QUERY_DEFAULTS } from 'store/constants';
+import sanitizeHTML from 'lib/sanitize-html';
+import withSelect from 'wc-api/with-select';
 
 class ReviewsPanel extends Component {
-	constructor( props ) {
-		super( props );
-		this.state = {
-			loading: true,
-			reviews: [],
-		};
-	}
-
-	componentDidMount() {
-		this.interval = setTimeout( () => {
-			this.setState( {
-				loading: false,
-				reviews: demoReviews,
-			} );
-		}, 5000 );
-	}
-
-	componentWillUnmount() {
-		clearTimeout( this.interval );
-	}
-
 	renderReview( review ) {
-		const product = demoProducts[ 0 ];
-
+		const product = review._embedded.up[ 0 ];
```

Comment:
Probably overly cautious, but do we want to check that these properties exist before rendering?